### PR TITLE
feat: adding code examples

### PR DESCRIPTION
**Pull request by Dudleyneedham**
## fixes https://github.com/KILTprotocol/ticket/issues/1438 &&  https://github.com/KILTprotocol/ticket/issues/1435 && https://github.com/KILTprotocol/ticket/issues/1436
Adding code examples

## How to test:

yarn install

yarn build

yarn start

## Checklist:

- [ ] I have verified that the code works
- [ ] I have verified that the code is easy to understand
  - [ ] If not, I have left a well-balanced amount of inline comments
- [ ] I have [left the code in a better state](https://deviq.com/principles/boy-scout-rule)
- [ ] I have documented the changes (where applicable)

### CODE REVIEW

File: docs/delegations.md [hunk 1]
```diff
@@ -93,14 +168,40 @@ Because transaction costs on the blockchain increase proportionally to the numbe
 
 However if the number of actual lookups performed is less than this number, excess funds will be returned after the transaction has completed. If this number is lower than the actual steps required, the transaction will fail.
 
-```
-    <CODE EXAMPLE: I revoke a regular attestation (max_depth=0) and delegated attestations with different numbers of lookups (max_depth=2 if I am the parent, max_depth=3 if I’m the parent’s parent>
+## Delegated Attestation
+
+![https://i.imgur.com/UmSNC1m.png](https://i.imgur.com/UmSNC1m.png)
```
Comment (weichweich) [inline docs/delegations.md]: this should go into our resource folder at https://github.com/KILTprotocol/docs/tree/master/static/img

File: docs/delegations.md [hunk 2]
```diff
@@ -93,14 +168,40 @@ Because transaction costs on the blockchain increase proportionally to the numbe
...
+
+
+CASE 1: The delegation node 3 revokes the attestation. The number of lookups performed will be zero as delegation node 3 created the delegatied attestation
+
+CASE 2: The delegation node 1 revokes the attestation. The number of lookups performed will be two as delegation node 3 created the delegated attestation, therefore, must count to find the delegation.
+
+Each CASE the delegator must call the revoke on the instantiated attestation object with the given delegator or parent of the delegator to revoke the attestation.
```
Comment (weichweich) [inline docs/delegations.md]: maybe we should tell the reader that this is an upper limit? So they can give more lookups and will get refunded the fees that they paid too much?

File: sidebars.js [hunk 3]
```diff
@@ -1,6 +1,12 @@
 module.exports = {
   someSidebar: {
-    Docusaurus: ['claiming', 'attesting', 'requesting-terms', 'verification', 'doc1', 'doc2', 'doc3'],
-    Features: ['mdx'],
```
Comment (weichweich) [inline sidebars.js:4]: do we want to disable `mdx`? had to look that up... https://mdxjs.com/ It looks uneccessary.

**Review by weichweich — CHANGES_REQUESTED** [review]
The image needs to go into our folder.

File: docs/delegations.md (hunk 1 above)
Comment (Dudleyneedham) [reply docs/delegations.md]: Should I add this to the resource folder? Since I can see it in your PR already?

File: sidebars.js (hunk 3 above)
Comment (Dudleyneedham) [reply sidebars.js:4]: Yes, sure. I will remove it.

File: docs/delegations.md (hunk 2 above)
Comment (Dudleyneedham) [reply docs/delegations.md]: I will add to the end of this. 
Each CASE the delegator must call the revoke on the instantiated attestation object with the given delegator or parent of the delegator to revoke the attestation. `The fee is taking considering the maximum number of lookups needed to reach all cases, these fees will be refunded if paid too much.`

File: sidebars.js (hunk 3 above)
Comment (weichweich) [reply sidebars.js:4]: ah no i was completely confused by this. It doesn't enable features! I thought you disabled the mdx

File: docs/delegations.md (hunk 1 above)
Comment (weichweich) [reply docs/delegations.md]: yeah already did it my self. 🙈 sorry

File: docs/delegations.md (hunk 2 above)
Comment (weichweich) [reply docs/delegations.md]: sounds good

**Comment by Dudleyneedham** [comment]
Please re-review

**Review by weichweich — APPROVED** [review]
Nice

**Pull request merged**